core/src/main/java/org/elasticsearch/index/mapper/geo/GeoPointFieldMapper.java

```diff
@@ -215,6 +216,7 @@ public GeoPointFieldMapper build(BuilderContext context) {
         @Override
         public Mapper.Builder<?, ?> parse(String name, Map<String, Object> node, ParserContext parserContext) throws MapperParsingException {
             Builder builder = geoPointField(name);
+            final boolean indexCreatedBefore2x = parserContext.indexVersionCreated().before(Version.V_2_0_0);
```

Comment:
I find the name a little confusing, since we already have this exact same name in FieldMapper, and it means before beta1. A better name would be good...I don't have a suggestion though...